action.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Easy Infra[structure as Code]'
```

Comment:
Can you point me to how you've tested this?  Thanks!

Reply:
Thanks for pointing that out. I added this action to another repo I invited you to - can you have a look at this action and let me know if I need to add chown/chmod or similar:
https://github.com/callmestu/Logging-AWS-AZ-Terraform/runs/5233640873